client/src/amplify-ui/cli-feature-flag/feature-flags.json

```
@@ -143,21 +143,105 @@
     "description": "Feature Flag related to Codegen",
     "features": {
       "useAppsyncModelgenPlugin": {
-        "description": "Allows to use migrated codegen package.",
+        "description": "Choose between implementations of internal DataStore model generator package. Refer [link to FeatureFlags.md]",
         "type": "Feature",
         "valueType": "Boolean",
         "versionAdded": "4.40.0",
         "deprecationDate": "May 1st 2021",
         "values": [
           {
             "value": "true",
-            "description": "Allows using newly migrated codegen package.",
+            "description": "[Recommended] Use the models generator internal package implementation at https://github.com/aws-amplify/amplify-codegen, which will be actively maintained",
```

Here and throughout: The flags still read oddly because of their reference to a point in time. Suggest we rewrite these to be simple declarations of behavior, with an up-front explicit note about deprecation of the old behavior.

```suggestion
            "description": "[Recommended] Use the models generator internal package implementation at https://github.com/aws-amplify/amplify-codegen",
```